SofaKernel/modules/SofaBaseLinearSolver/CGLinearSolver.inl

```diff
@@ -231,6 +251,20 @@ void CGLinearSolver<TMatrix,TVector>::solve(Matrix& M, Vector& x, Vector& b)
             break;
         }
 
+         if( fabs(den) == 0.0 )
+         {
+             endcond = "div by zero. You scene is probably buggy.";
+             if( verbose )
+             {
+                 sout<<"CGLinearSolver, den = "<<den<<", smallDenominatorThreshold = "<<f_smallDenominatorThreshold.getValue()<<sendl;
+             }
```

Comment:
Of  course, if you think it is relevant you can fix it in another PR but please keep in mind that if everyone open PR to fix scenes it will be a big mess very quickly and we will have to "ping pong" between all the PRs.